### DISCUSSION TITLE
Using Harmony with Multi-Ome

### DISCUSSION THREAD

**Discussion by waltno** [GreenleafLab/ArchR#1193] [Questions / Documentation] [answered]
I am wondering what is the best way to use Harmony to correct batch effects with multi-ome data. Should I add harmony for both ATAC/RNA assays and Harmony again with combined dimensions? Or run harmony at the very end?

**Comment by rcorces** [comment, marked as answer]
I'm not sure there is a "right" answer to this. I dont think I would run Harmony 3 times (both atac/rna and again for combined). I think I would either run it once for both ATAC and RNA prior to making the combined dims or just once after making the combined dims.

**Reply by sylestiel** [reply]
@rcorces 

When working with multiple multiome datasets, if I choose to run Harmony after the step
proj <- addCombinedDims(proj, reducedDims = c("LSI_ATAC", "LSI_RNA"), name =  "LSI_Combined")

1) How can I generate/render Harmony based embedding using LSI_ATAC and LSI_RNA in addition

2) How do I specify getMarkerFeatures based on Harmony correction?
